common/button/button-en.html

```diff
@@ -0,0 +1,426 @@
+<div class="btn-group" role="group" aria-label="First group">
+<button type="button" class="btn btn-default">1</button>
+<button type="button" class="btn btn-default">2</button>
+<button type="button" class="btn btn-default">3</button>
+<button type="button" class="btn btn-default">4</button>
+</div>
+<div class="btn-group" role="group" aria-label="Second group">
+<button type="button" class="btn btn-default">5</button>
+<button type="button" class="btn btn-default">6</button>
+<button type="button" class="btn btn-default">7</button>
+</div>
+<div class="btn-group" role="group" aria-label="Third group">
+<button type="button" class="btn btn-default">8</button>
+</div>
+</div>
+
+<h4>Code sample</h4>
+<pre><code>// Button toolbar:
+&lt;div class="<strong>btn-toolbar</strong>" role="toolbar" aria-label="Toolbar with button groups"&gt;
+	&lt;div class="btn-group" role="group" aria-label="First group"&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		1
+		&lt;/button&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		2
+		&lt;/button&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		3
+		&lt;/button&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		4
+		&lt;/button&gt;
+	&lt;/div&gt;
+
+	&lt;div class="btn-group" role="group" aria-label="Second group"&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		5
+		&lt;/button&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		6
+		&lt;/button&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		7
+		&lt;/button&gt;
+	&lt;/div&gt;
+
+	&lt;div class="btn-group" role="group" aria-label="Third group"&gt;
+		&lt;button type="button" class="btn btn-default"&gt;
+		8
+		&lt;/button&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3 id="inputgroups">Input Groups</h3>
+<p>Easily extend form controls by adding buttons, or button groups on either side of textual inputs with the <code>input-group</code> and <code>input-group-btn</code> classes.</p>
+<h4>Working Example</h4>
+<div class="input-group mb-3">
+<div class="input-group-btn">
+<button class="btn btn-default" type="button">Button</button>
+</div>
+<input type="text" class="form-control">
+</div>
+
+<div class="input-group mb-3">
+<input type="text" class="form-control">
+<div class="input-group-btn">
+<button class="btn btn-default" type="button">Button</button>
+</div>
+</div>
+
+<h4>Code Sample</h4>
+<pre><code>// Input groups:
+&lt;div class="<strong>input-group</strong> mb-3"&gt;
+	&lt;div class="<strong>input-group-btn</strong>"&gt;
+		&lt;button class="btn btn-default" type="button"&gt;Button&lt;/button&gt;
+	&lt;/div&gt;
+	&lt;input type="text" class="form-control"&gt;
+&lt;/div&gt;
+
+&lt;div class="<strong>input-group</strong> mb-3"&gt;
+	&lt;input type="text" class="form-control"&gt;
+	&lt;div class="<strong>input-group-btn</strong>"&gt;
+		&lt;button class="btn btn-default" type="button"&gt;Button&lt;/button&gt;
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
```

Comment:
Remove the class `mb-3`

And considering the use of this style is more for phrasing content, use `<span>` instead of `<div>` as proposed bellow

```suggestion
<div class="input-group">
<span class="input-group-btn">
<button class="btn btn-default" type="button">Button</button>
</span>
<input type="text" class="form-control">
</div>

<div class="input-group">
<input type="text" class="form-control">
<span class="input-group-btn">
<button class="btn btn-default" type="button">Button</button>
</span>
</div>

<h4>Code Sample</h4>
<pre><code>// Input groups:
&lt;div class="<strong>input-group</strong>"&gt;
	&lt;span class="<strong>input-group-btn</strong>"&gt;
		&lt;button class="btn btn-default" type="button"&gt;Button&lt;/button&gt;
	&lt;/span&gt;
	&lt;input type="text" class="form-control"&gt;
&lt;/div&gt;

&lt;div class="<strong>input-group</strong>"&gt;
	&lt;input type="text" class="form-control"&gt;
	&lt;span class="<strong>input-group-btn</strong>"&gt;
		&lt;button class="btn btn-default" type="button"&gt;Button&lt;/button&gt;
	&lt;/span&gt;
&lt;/div&gt;</code></pre>
```